polaris.shopify.com/content/tools/polaris-migrator.mdx

```diff
@@ -37,35 +37,6 @@ npx @shopify/polaris-migrator <migration> <path>
 
 ## Migrations
 
-### @shopify/polaris-react v14.0.0
-
-If you are upgrading Polaris from v13 to v14 please follow our [migration guide](/version-guides/migrating-from-v13-to-v14).
```

Comment:
The only line I'd remove here is this one and then make sure to add it to the v14 branch.